src/test/java/edu/harvard/iq/dataverse/api/HarvestingClientsIT.java

```diff
@@ -299,6 +299,11 @@ private void harvestingClientRun(boolean allowHarvestingMissingCVV)  throws Inte
         }
```

Comment:
Hmm, it's been so long that https://jenkins.dataverse.org/job/IQSS-Dataverse-Develop-PR/job/PR-9013/7/display/redirect shows a failure but that job is now a 404 so I can't see any details.

After you merge the latest from develop let's keep an eye on the new Jenkins run.

Reply:
Last time all checks were green except for "continuous-integration/jenkins/pr-merge " that was pending. I'm not clear on what it is doing.

Reply:
Ok! Now that the app is being deployed following the SQL script renaming, I can see that a couple harvesting-related tests are failing:

- edu.harvard.iq.dataverse.api.HarvestingClientsIT.testHarvestingClientRun_AllowHarvestingMissingCVV_False(HarvestingClientsIT.java:187)
- edu.harvard.iq.dataverse.api.HarvestingClientsIT.testHarvestingClientRun_AllowHarvestingMissingCVV_True(HarvestingClientsIT.java:191)

@plecor can you please take a look? Do you need help with how to run these tests?

Reply:
Hi @pdurbin ,

I've re-run the tests from scratch and I wonder if it could be a chicken and egg situation. The new assertion fails if I run the tests from this branch because the sql migration file it's adding hasn't been run. If I manually update the db then the tests run fine. Could it be something similar going on with Jenkins?

Reply:
Ok, the issue is that in a new install, the corresponding data is initialized by `afterMigrate__1-7256-upsert-referenceData.sql` after the other migrations, so the data we're trying to update doesn't exist yet.

I updated this file with the same change as the sql migration file.

Reply:
Interesting. Thanks.

@poikilotherm judging from https://github.com/IQSS/dataverse/blame/c8499ba9553ac46cf3adc56d1b9e56f0c781d30f/src/main/resources/db/migration/afterMigrate__1-7256-upsert-referenceData.sql you added that "after migrate" file. What do you think? Any risk in changing it? 🤔

Reply:
@qqmyers took a look and had this to [say](https://iqss.slack.com/archives/C03R1E7T4KA/p1733848500273549):

"that afterMigrate script appears to run after everything else, every time you restart, rather than being one of the scripts tracked by hash value in the flyway_schema_history table. So - it looks OK to me to update it. (Mostly from looking at https://documentation.red-gate.com/fd/callback-concept-184127466.html and related and verifying that I don't see it in the flyway_schema_history table)."